### PR TITLE
Add contribution guidelines

### PR DESCRIPTION
**Pull request by marinavega**
Connects to #3  .

We should add both PR and issue template links whenever possible.

### CODE REVIEW

File: en/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,3 +1,50 @@
...
-Please note that this project is released with a [Contributor Code of Conduct](../CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms.
+:sparkles::heart: Thank you so much for taking the time to contribute! :heart::sparkles:
+
+The following is a set of guidelines for contributing to any of our projects, which are hosted in the [FolksWhoCode Organization](https://github.com/folkswhocode) on GitHub. Feel free to propose changes to this document in a pull request.
+
+#### Table Of Contents
+
+[Code of Conduct and Values](#code-of-conduct-and-values)
+
+[How Can I Contribute?](#how-can-i-contribute)
```
Comment (dreamingechoes) [inline en/CONTRIBUTING.md]: We have to fix these list elements: first and fourth elements are not indented and they don't have the `*` character to make them a markdown list element 😄

File: en/CONTRIBUTING.md [hunk 2]
```diff
@@ -1,3 +1,50 @@
...
+## How Can I Contribute?
+
+When contributing to this repository, please first discuss the change you wish
+to make via issue :smile:
+
+### Your First Contribution
+
+You can start by looking through these `beginner-friendly`, `good-first-issue` and `help-wanted` issues:
+
+* [Beginner issues][beginner-friendly]
```
Comment (dreamingechoes) [inline en/CONTRIBUTING.md]: Maybe it's good to complete these example labels with some information about what they mean, right? 😄

Comment (marinavega) [reply en/CONTRIBUTING.md]: Sure. I'll get right on it 💪

**Comment by dreamingechoes** [comment]
Cool! Let's merge this PR! 👏 😄 💖

**Pull request merged**